pkg/gofer/origins/upbit_test.go

```diff
@@ -158,16 +159,16 @@ func (suite *UpbitSuite) TestSuccessResponse() {
 }
 
 func (suite *UpbitSuite) TestRealAPICall() {
-	testRealAPICall(suite, &Upbit{Pool: query.NewHTTPWorkerPool(1)}, "ETH", "BTC")
-	pairs := []Pair{
-		{Base: "KNC", Quote: "KRW"},
-		{Base: "MANA", Quote: "KRW"},
-		{Base: "OMG", Quote: "KRW"},
-		{Base: "REP", Quote: "KRW"},
-		{Base: "SNT", Quote: "KRW"},
-		{Base: "ZRX", Quote: "KRW"},
-	}
-	testRealBatchAPICall(suite, &Upbit{Pool: query.NewHTTPWorkerPool(1)}, pairs)
+	//testRealAPICall(suite, &Upbit{Pool: query.NewHTTPWorkerPool(1)}, "ETH", "BTC")
```

Comment:
Why this comments?

Reply:
Ouch, I uncommented all except this one. Good catch !